### PR TITLE
Improve documentation of tutorial part-eight

### PR DESCRIPTION
**Pull request by davidmpaz**
Hi,

This PR state explicitly whats need to be done, in the context of the tutorial, to finalize it including all features explained.

Maybe it is a bit picky but, still for a no brainer experience it could help people like me :-D

### CODE REVIEW

File: docs/tutorial/part-eight/index.md [hunk 1]
```diff
@@ -190,6 +190,28 @@ class Application extends React.Component {
 
 > 💡 The above example is from the [React Helmet docs](https://github.com/nfl/react-helmet#example). Check those out for more!
 
+If you got confused by our last example reference to `class Application extends React.Component`, dont be!
+
+To make it work Helmet in your blog, add the `Helmet` component in your `src/pages/index.js` file like:
```
Comment (KirankumarAmbati) [inline docs/tutorial/part-eight/index.md]: May be you have to change **"To make it work Helmet in your blog"** to **"To make it work in your blog"** or **"To make Helmet work in your blog"** based on the context. Please verify and change accordingly.

Comment (davidmpaz) [reply docs/tutorial/part-eight/index.md]: Done. Thanks for the feedback!

Comment (KirankumarAmbati) [reply docs/tutorial/part-eight/index.md]: Thanks for quick action ! 👍

**Comment by calcsam** [comment]
Thanks for submitting this. It seems this is trying to do two things. Explain the `extends` terminology and further explain `Helmet`. Unfortunately these aren't helping the reader get through the document, so I'll close this PR. What we need here is support for annotations, which we don't have now.

**Comment by shannonbux** [comment]
Hi @davidmpaz! Can you clarify what problem you are trying to solve? Was there a step missing from the tutorial?

**Comment by davidmpaz** [comment]
Hi @shannonbux, 

it is all about how to declaring the helmet component in part eight `Use React Helmet in your pages:` there, an example of another application is made:
```
class Application extends React.Component {
  render() {
    return (
      <div className="application">
        <Helmet>
          <meta charSet="utf-8" />
          <title>My Title</title>
          <link rel="canonical" href="http://mysite.com/example" />
        </Helmet>
        ...
      </div>
    )
  }
}
```

 for a no brainer reading experience my sugestion is to make Helmet work in the blog example in `src/pages/index.js` file of that part of the tutorial, just to follow with the idea of consistency the whole tutorial had followed.

Like i said before, this could look picky and too detailed but it can help newbies like me who are exploring new tools.

thanks for coming back to this

**Comment by shannonbux** [comment]
Hi @davidmpaz! Thanks for the details! I will be doing usability testing for all parts of the tutorial at some point in the next few months and will keep an eye out for whether this is a common problem.

**Pull request closed without merging**